src/components/main-dashboard/DailyWeather.jsx

```diff
@@ -11,7 +11,8 @@ export default function DailyWeather() {
       const { list } = dailyWeather;
       const dailyWeatherInfo = list.map((weatherData, index) => {
         const weatherObj = {
-          id: index,
+          key: index,
+          id: weatherData.weather[0].id,
```

Comment:
This is identical to the iconKey value right under maxTemperature. Is there a reason you pass create a id key?